wikimedia/wikipedia-ios
# Donor experiences translations

**Phabricator:** 
https://phabricator.wikimedia.org/T288285

### Notes
This PR adds final translations manually from our donor experiences translations [spreadsheet](https://docs.google.com/spreadsheets/d/1EXzgDZizHFnJsr1G5r4SU4Rp0l1ouQk9T3jqLGKb9jU/edit#gid=0). I will update these changes in Translatewiki after this is merged.

### Test Steps
1. Change device region to NL or IT. Change device language to NL or IT.
2. Add test card in Apple Wallet. Run app. Pull to refresh on Explore feed to fetch campaigns.
3. Confirm Apple Pay donate form is properly translated when device language is IT or NL.

### Screenshots

![IMG_0258](https://github.com/wikimedia/wikipedia-ios/assets/3620196/8cdf0e0f-864f-4d82-9f04-01d674fcd6d3)
![IMG_0259](https://github.com/wikimedia/wikipedia-ios/assets/3620196/e2c0c11f-611f-4ec1-b7ef-082000e7c8be)
![IMG_0261](https://github.com/wikimedia/wikipedia-ios/assets/3620196/298492df-8b03-4800-8dcb-33bf7038c446)
![IMG_0262](https://github.com/wikimedia/wikipedia-ios/assets/3620196/29764988-b6fc-4231-bca9-75d8824979f9)
![IMG_0263](https://github.com/wikimedia/wikipedia-ios/assets/3620196/a60f699c-71d1-4566-986c-842fdf5f65cd)
![IMG_0264](https://github.com/wikimedia/wikipedia-ios/assets/3620196/b628964b-994a-4920-a434-e8582ff38eb8)
![IMG_0265](https://github.com/wikimedia/wikipedia-ios/assets/3620196/004245d4-e584-4d5b-a4ad-b7bd567bfe1d)
![IMG_0266](https://github.com/wikimedia/wikipedia-ios/assets/3620196/d54a9f82-6059-4d36-8377-897dda560cea)
